sphinx_gallery/_static/gallery.css

```diff
@@ -55,7 +55,9 @@ div.sphx-glr-thumbContainer {
 
 .sphx-glr-script-out pre {
   overflow-y: auto;
-  max-height: 30em; }
+  max-height: 30em;
+  background-color: #FFFFFE;
```

Comment:
This is almost white and I thought you wanted it light yellow ... did you mean #FFFFEF instead?